testing/xfpga/test_sysfs_c.cpp

```diff
@@ -832,19 +838,87 @@ TEST_P(sysfs_c_mock_p, fpga_sysfs_07) {
 *            When passed with invalid argument return
 *            FPGA_INVALID_PARAM <br>
 */
-TEST_P(sysfs_c_mock_p, fpga_sysfs_08) {
+TEST_P(sysfs_dfl_c_mock_p, fpga_sysfs_08) {
 	fpga_result result;
 	char sysfs_path[SYSFS_PATH_MAX] = { 0 };
 
-	result = sysfs_get_fme_perf_path(handle_, sysfs_path);
+	result = sysfs_get_fme_perf_path(tokens_[0], sysfs_path);
+	EXPECT_EQ(result, FPGA_NOT_FOUND);
```

Comment:
Why does this return FPGA_NOT_FOUND? Not supported in dfl 2? description doesn't match test.

Reply:
fixed